File: C/PSET1/cash.c
#include<cs50.h>
#include<stdio.h>
#include <math.h>
int main(void)
{
    
    float input;
    //prompt user for input
    do{
        input = get_float("input: ");
    }while(input<0);
    //convert from float to int 
    int coins = round(input * 100)  ;    
    
    int output = 0;
    //calculate output
    while(coins>0)
    {
        if((coins / 25) > 0) 
        {
            coins -= 25;
            output++;
        }
        else if((coins / 10) > 0)
        {
            coins -= 10;
            output++;
        }
        else if((coins / 5) > 0)
        {
            coins -= 5;
            output++;
        }
        else
        {
            coins -= 1;
            output++;
        }
        
    }

printf("%d\n",output);
}

File: Python/Cash/cash.py
from cs50 import get_float

input = 0
while True:
    input = get_float("input: ")
    if input > 0:
        break

coins = round(input * 100, 0)

output = 0

while coins > 0:
    
    if (coins >= 25):
        coins -= 25
        output += 1

    elif (coins >= 10):
        coins -= 10
        output += 1

    elif(coins >= 5):
        coins -= 5
        output += 1

    else:
        coins -= 1
        output += 1
print(output)
